s4e-backend/src/main/java/pl/cyfronet/s4e/bean/Invitation.java

```diff
@@ -0,0 +1,41 @@
+package pl.cyfronet.s4e.bean;
+
+import lombok.*;
+import pl.cyfronet.s4e.bean.audit.CreationAndModificationAudited;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+@Entity
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "invitation", uniqueConstraints = @UniqueConstraint(columnNames = {"email", "institution_id"}))
+public class Invitation extends CreationAndModificationAudited {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @EqualsAndHashCode.Include
```

Comment:
By default, `@EqualsAndHashCode` (inherited in this class through `@Data`) include in its generated methods all fields, except the ones annotated with `@EqualsAndHashCode.Exclude`, so you don't have to add `Include` on every field.